osdk/tools/build_image.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# SPDX-License-Identifier: MPL-2.0
+
+set -e
+
+SCRIPT_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+ASTER_ROOT_DIR=${SCRIPT_DIR}/../..
+ASTER_RUST_VERSION=$( grep -m1 -o 'nightly-[0-9]\+-[0-9]\+-[0-9]\+' ${ASTER_ROOT_DIR}/rust-toolchain.toml )
+VERSION=$( cat ${ASTER_ROOT_DIR}/VERSION )
+IMAGE_NAME=asterinas/osdk:${VERSION}
```

Comment:
Since Asterinas Kernel is based on OSDK, the Docker image of the Asterinas Kernel should be based on the OSDK's Docker Image, right?